services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -133,4 +134,15 @@ data class Hanke(
     fun toLogString(): String {
         return toString()
     }
+
+    override fun extractYhteystiedot(): List<HankeYhteystieto> =
```

Comment:
Are these two overriding functions necessary or would the default implementation in the interface be sufficient?

Reply:
I tried removing it, but the audit logging functions expect to get HankeYhteystieto from this method. And I don't feel comfortable changing them to use the interface instead.

Reply:
Fair enough.

Reply:
`yhteystiedotByType()` was redundant. I removed it.